ui/raidboss/data/07-dt/hunts/living_memory.ts

```diff
@@ -245,14 +332,208 @@ const triggerSet: TriggerSet<Data> = {
         next: Outputs.next,
       },
     },
+
+    // ****** Boss Fate: Mica the Magical Mu ****** //
+    // Aoe (9733 is cast shortly later as the actual aoe)
+    {
+      id: 'Hunt Mica the Magical Mu Spark of Imagination',
+      type: 'StartsUsing',
+      netRegex: { id: '9732', source: 'Mica the Magical Mu', capture: false },
+      response: Responses.aoe(),
+    },
+    // tank buster cleave (part of 9730 Shimmerstrike)
+    {
+      id: 'Hunt Mica the Magical Mu Shimmerstorm Tankbuster',
+      type: 'StartsUsing',
+      netRegex: { id: '9731', source: 'Mica the Magical Mu', capture: true },
+      suppressSeconds: 1,
+      response: Responses.tankCleave(),
+    },
+    // random puddles under people. Shimmerstorm (972F) or Shimmerstrike (9730) spawn them.
+    {
+      id: 'Hunt Mica the Magical Mu Shimmerstorm',
+      type: 'StartsUsing',
+      netRegex: { id: '972F', source: 'Mica the Magical Mu', capture: false },
+      suppressSeconds: 1,
+      response: Responses.moveAway(),
```

Comment:
I think this would be clearer with a "dodge puddles" or some similar callout, rather than the generic "Move!", since the puddles do not always require dodging depending on where they appear.